tally_ho/apps/tally/views/data/tally_list_view.py

```diff
@@ -29,6 +36,15 @@ def render_column(self, row, column):
         else:
             return super(TallyListDataView, self).render_column(row, column)
 
+    def filter_queryset(self, qs):
+        keyword = self.request.GET.get('search[value]', None)
+
+        if keyword:
+            qs = qs.filter(Q(name__contains=keyword) |
+                           Q(id__contains=keyword))
```

Comment:
Do we want to use `icontains` instead of `contains`?

Reply:
Cool i think we should use `icontains` because it's case insensitive

Reply:
Done